index.js

```diff
@@ -100,12 +100,16 @@ module.exports = {
     },
     storage: {
         metadata: {
-            MetadataFileServer:
-            require('./lib/storage/metadata/file/MetadataFileServer'),
-            MetadataFileClient:
-            require('./lib/storage/metadata/file/MetadataFileClient'),
-            LogConsumer:
-            require('./lib/storage/metadata/bucketclient/LogConsumer'),
+            bucketclient: {
```

Comment:
Not sure your thoughts on using typescript but this will for sure conflict with the existing TS PRs.